src/target.d

```diff
@@ -199,8 +199,13 @@ struct Target
         }
     }
 
-    /*
-     * Return true if the given type is supported for this target
+    /**
+     * Checks whether the target supports a vector type with total size `sz`
+     * (in bytes) and element type `type`.
+     *
+     * Returns: 0 if the type is supported, or else: 1 if vector types are not
+     *     supported on the target at all, 2 if the given size isn't, or 3 if
+     *     the element type isn't.
```

Comment:
Sorry, I meant adding an additional value (like 4) to indicate a possibility for emulation. (For a moment I thought that the max value was 2 and that's why I said the next available - 3).

This PR is obviously uncontroversial and should be merged.

This discussion is probably more appropriate for dmd-internal.